guides/keras_cv/retina_net_overview.py

```diff
@@ -0,0 +1,275 @@
+"""
+Title: Train an Object Detection Model on Pascal VOC 2007 using KerasCV
+Author: [lukewood](https://lukewood.xyz)
+Date created: 2022/08/02
+Last modified: 2022/08/02
+Description: Use KerasCV to train a RetinaNet on Pascal VOC 2007.
+"""
+
+"""
+## Overview
+
+KerasCV offers a complete set of APIs to allow you to train your own state-of-the-art,
+production-grade object detection model.  These APIs include object detection specific
+data augmentation techniques, models, and COCO metrics.
+
+To get started, let's sort out all of our imports and define global configuration parameters.
+"""
+
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow import keras
+from tensorflow.keras import callbacks as callbacks_lib
```

Comment:
This is only used 3x in 1 location, I'd recommend just accessing `keras.callbacks` instead of importing the submodule